modules/serverless-ossm-installing-and-configuring-openshift-serverless.adoc

```diff
@@ -0,0 +1,201 @@
+:_content-type: PROCEDURE
+[id="serverless-ossm-installing-and-configuring-openshift-serverless_{context}"]
+= Installing and configuring {ServerlessProductShortName}
+
+After installing {SMProductShortName}, you need to install {ServerlessProductShortName} with a specific configuration.
+
+.Procedure
+
+. Install Knative Serving with the following `KnativeServing` custom resource, which enables the Istio integration:
++
+.Example `knative-serving-config.yaml` configuration file
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  ingress:
+    istio:
+      enabled: true <1>
+  deployments: <2>
+  - name: activator
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: autoscaler
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  config:
+    istio: <3>
+      gateway.knative-serving.knative-ingress-gateway: istio-ingressgateway.<your-istio-namespace>.svc.cluster.local
+      local-gateway.knative-serving.knative-local-gateway: knative-local-gateway.<your-istio-namespace>.svc.cluster.local
+----
+<1> Enable Istio integration.
+<2> Enable sidecar injection for Knative Serving data plane pods.
+<3> If your istio is not running in the `istio-system` namespace, you need to set these two flags with the correct namespace.
+
+. Apply the `KnativeServing` resource:
++
+[source,terminal]
+----
+$ oc apply -f knative-serving-config.yaml
+----
+
+. Install Knative Eventing with the following `KnativeEventing` custom resource, which enables the Istio integration:
```

Comment:
```suggestion
. Install Knative Eventing with the following `KnativeEventing` object, which enables the Istio integration:
```
Again, just for consistency. "Custom resource" isn't really necessary to call out here, I don't think.